assets/resource.h

```diff
@@ -0,0 +1 @@
+#define IDI_APPICON 101
```

Comment:
Having a define in a separate header that is used only once seems a bit overkill to me. The define itself is okay since it explains the magic value, I just don't get the extra header.